tools/structure/src/config/properties.json

```diff
@@ -2,7 +2,7 @@
   "schemaVersion": "0.0.1",
   "displayName": "Structure Tool",
   "functionName": "structure_tool",
-  "toolVersion": "0.0.28",
+  "toolVersion": "0.0.30",
```

Comment:
why it jumped `28` to `30` . Are we missed `29`?

Reply:
Who ever rolled to 0.0.29 [in cloud-platform-registry](https://github.com/Zipstack/unstract-cloud-platform/blob/4cc03ad1547f0b009e851f7e5ba83c86b42a27c7/docker/sample.backend.env#L94) did not make this change in OSS. Hence the confusion @muhammad-ali-e